FlashCardApp/DTO/StudyAreaDTO.cs

```diff
@@ -0,0 +1,7 @@
+namespace FlashCardApp.DTO;
+
+public class StudyAreaDTO
+{
+    public DateTime Date { get; set; }
+    public int Score { get; set; }
```

Comment:
I think here we needed to add which stack I've studied in each study session.

Reply:
Alrighty, I'll make the changes and create a new PR